worlds/ufo50/barbuta/items.py

```diff
@@ -0,0 +1,46 @@
+from typing import TYPE_CHECKING, Dict, NamedTuple, List
+from BaseClasses import Item, ItemClassification as IC
+from .locations import barbuta_base_id
+
+if TYPE_CHECKING:
+    from .. import UFO50World
+
+
+class BarbutaItem(Item):
+    game: str = "UFO 50"
+
+
+class ItemInfo(NamedTuple):
+    id_offset: int
+    classification: IC
+    quantity: int
+
+
+barbuta_item_table: Dict[str, ItemInfo] = {
+    "$50": ItemInfo(0, IC.progression, 5),
+    "$100": ItemInfo(1, IC.progression, 5),
+    "Umbrella": ItemInfo(2, IC.progression, 1),
+    "Necklace": ItemInfo(3, IC.progression, 1),
+    "Pin": ItemInfo(4, IC.progression | IC.useful, 1),
+    "Candy": ItemInfo(5, IC.progression, 1),
+    "Wand": ItemInfo(6, IC.progression | IC.useful, 1),
+    "Blood Sword": ItemInfo(7, IC.progression, 1),
+    "Key": ItemInfo(8, IC.progression, 1),
+    "Bat Soul": ItemInfo(9, IC.progression, 1),
```

Comment:
The internal name for this seems to be "Bat Orb" but yours sounds cooler lol

Reply:
Eh, I'll swap it to the internal name. Less confusing that way